### ISSUE TITLE
Comments lack contrast between link colors and the background

### ISSUE THREAD

**Issue by essential-randomness** [BobaBoard/boba-components#77]
We have in the past tried to find a color that fit accessibility standards and failed. Regardless let's make the situation better and reevaluate our whole theme color scheme at some point in the future.
